lib/go-tc/deliveryservices.go

```diff
@@ -198,7 +198,6 @@ type DeliveryServiceNullableV11 struct {
 	//
 	Active                   *bool                   `json:"active" db:"active"`
 	AnonymousBlockingEnabled *bool                   `json:"anonymousBlockingEnabled" db:"anonymous_blocking_enabled"`
-	CacheURL                 *string                 `json:"cacheurl" db:"cacheurl"`
```

Comment:
This breaks backward compatibility at a code level. Shouldn't this stay present but empty here and only be removed in later versions of the type? As it is, code that uses the V11 struct will break.

Reply:
You are correct, I did not mean to commit this change.